scrapy/extensions/batches.py

```diff
@@ -0,0 +1,73 @@
+"""
+Extensions for batch processing and support.
+"""
+
+
+class Batch:
```

Comment:
I think `Batch` is a bad name, since this class does not represent a single batch. Maybe something in the line of `BatchTracker` or `BatchHandler`?

Reply:
I was having a bad day naming variables :D. I think `BatchHandler` would do nicely.